jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/CreditResource.java

```diff
@@ -121,23 +120,24 @@ public Response createCredit(final CreditJson json,
                                  @javax.ws.rs.core.Context final UriInfo uriInfo) throws AccountApiException, InvoiceApiException {
         verifyNonNullOrEmpty(json, "CreditJson body should be specified");
         verifyNonNullOrEmpty(json.getAccountId(), "CreditJson accountId needs to be set",
-                             json.getCreditAmount(), "CreditJson creditAmount needs to be set");
+                             json.getAmount(), "CreditJson creditAmount needs to be set");
 
         final Iterable<PluginProperty> pluginProperties = extractPluginProperties(pluginPropertiesString);
         final CallContext callContext = context.createCallContextWithAccountId(json.getAccountId(), createdBy, reason, comment, request);
 
         final Account account = accountUserApi.getAccountById(json.getAccountId(), callContext);
         final LocalDate effectiveDate = new LocalDate(callContext.getCreatedDate(), account.getTimeZone());
 
+        // TODO Having a common method is great, but the validation is quite minimal...
```

Comment:
Related to our discussion from yesterday: is that really a TODO (i.e. do we want to do anything)?